.policy.yml

```diff
@@ -0,0 +1,48 @@
+policy:
+  approval:
+  - or:
+    - Dependabot update
+    - Code change
+
+approval_rules:
+- name: Dependabot update
+  requires:
+    count: 1
+    teams:
+    - "coopnorge/engineering"
+  options:
+    invalidate_on_push: true
+    request_review:
+      enabled: true
+      mode: all-users
+    methods:
+      github_review: true
+  if:
+    has_successful_status:
+      - "build"
+    only_has_contributors_in:
+      users:
+      - "dependabot[bot]"
+    only_changed_files:
+      paths:
+      - "^go.mod$"
+      - "^go.sum$"
+      - "^.github/workflows/*"
+    has_valid_signatures_by_keys:
+      key_ids: ["4AEE18F83AFDEB23"]
```

Comment:
We might be able to make this into a shared rule: https://github.com/palantir/policy-bot#configuration

> The file is read from the most recent commit on the target branch of each pull request.
>
> The file may contain a reference to a policy in a different repository (see [Remote Policy Configuration](https://github.com/palantir/policy-bot#remote-policy-configuration).)
>
> If the file does not exist in the repository, policy-bot tries to load a shared policy.yml file at the root of the .github repository in the same organization. You can change this path and repository name when running your own instance of the server.
>
>If a policy does not exist in the repository or in the shared organization repository, policy-bot does not post a status check on the pull request. This means it is safe to enable policy-bot on all repositories in an organization.

But lets get something that works first

Reply:
yeah shared rules can be very useful.

Reply:
Shall we do that now, or promote later when we have a bit more experience?

Reply:
no later i would guess

Reply:
It makes sense to keep it local for a short while, but I hope the experience is good enough to escalate to org level quickly

Reply:
then we keep it like this for now